### PR TITLE
Create Fixed NavBar

### PR DESCRIPTION
**Pull request by yousefsayedd1**
Issue #14

### CODE REVIEW

**Comment by behnamazimi** [comment]
You may want to use `position:sticky` to get a similar result. read more [here](https://developer.mozilla.org/en-US/docs/Web/CSS/position)

**Comment by yousefsayedd1** [comment]
@behnamazimi  Yes, it is better, I edited it

File: 02.lovely-movies/styles/styles.css [hunk 1]
```diff
@@ -74,6 +74,10 @@ body.loading .loader {
     align-items: center;
     box-shadow: 0 1px 10px 2px rgba(0, 0, 0, 0.3);
     height: 70px;
+    position: sticky;
+    top: 0;
+    z-index: 5;
```
Comment (behnamazimi) [inline 02.lovely-movies/styles/styles.css:79]: I think `1` would be enough as the value.

File: 02.lovely-movies/styles/styles.css [hunk 2]
```diff
@@ -74,6 +74,10 @@ body.loading .loader {
...
+
```
Comment (behnamazimi) [inline 02.lovely-movies/styles/styles.css:80]: Unnecessary change.

File: 02.lovely-movies/styles/styles.css [hunk 3]
```diff
@@ -117,6 +121,7 @@ body.loading .loader {
 
 
 /** Main Content - START **/
+
```
Comment (behnamazimi) [inline 02.lovely-movies/styles/styles.css:124]: Unnecessary change.

File: 02.lovely-movies/styles/styles.css (hunk 1 above)
Comment (yousefsayedd1) [reply 02.lovely-movies/styles/styles.css:79]: but I added it in .main-header not  the body, is there some thing I don't understand.
image

Comment (behnamazimi) [reply 02.lovely-movies/styles/styles.css]: Sorry, may bad!

**Review by behnamazimi — APPROVED** [review]
